environments/01-network/test.tfvars

```diff
@@ -0,0 +1,38 @@
+enable_debug = "true"
+
+network_address_space                  = "10.48.64.0/18"
+aks_00_subnet_cidr_blocks              = "10.48.16.0/20"
+aks_01_subnet_cidr_blocks              = "10.48.32.0/20"
+iaas_subnet_cidr_blocks                = "10.48.1.0/24"
+application_gateway_subnet_cidr_blocks = "10.48.0.128/25"
+
+additional_subnets = [
+]
+
+private_dns_subscription = "1497c3d7-ab6d-4bb7-8a10-b51d03189ee3"
+private_dns_zones = [
+  "perftest.platform.hmcts.net",
+  "service.core-compute-perftest.internal",
+  "service.core-compute-idam-perftest.internal",
+  "service.core-compute-perftest.internal",
+  "service.core-compute-idam-perftest.internal",
```

Comment:
This does not reflect the current private_dns_zones in ARM
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/templates/vars/vnet/perftest.json#L11

Reply:
updated and also corrected the right subscription.